apps/proxy-auth/src/app/otp/send-otp/send-otp.component.ts

```diff
@@ -63,7 +63,7 @@ export class SendOtpComponent extends BaseComponent implements OnInit, OnDestroy
     public otpWidgetData;
     public showRegistration = new BehaviorSubject<boolean>(false);
     public referenceElement: HTMLElement = null;
-
+    public showLogin: boolean = false;
```

Comment:
this was supposed to be mapped with service subject
`public showLogin: BehaviorSubject<boolean> = this.otpWidgetService.showlogin;`